tests/Rules/Deprecations/CallToDeprecatedStaticMethodRuleTest.php

```diff
@@ -53,10 +53,6 @@ public function testDeprecatedStaticMethodCall(): void
 					'Call to deprecated method deprecatedFoo2() of class CheckDeprecatedStaticMethodCall\Foo.',
 					13,
 				],
-				[
-					'Call to deprecated method deprecatedFoo() of class CheckDeprecatedStaticMethodCall\Foo.',
```

Comment:
Because of

https://github.com/phpstan/phpstan-deprecation-rules/blob/1ed86a547d50ea916eeccb664abc6e9979bcf7c9/src/Rules/Deprecations/CallToDeprecatedStaticMethodRule.php#L43-L45

and

https://github.com/phpstan/phpstan-deprecation-rules/blob/1ed86a547d50ea916eeccb664abc6e9979bcf7c9/tests/Rules/Deprecations/data/call-to-deprecated-static-method-definition.php#L13-L19

and

https://github.com/phpstan/phpstan-deprecation-rules/blob/1ed86a547d50ea916eeccb664abc6e9979bcf7c9/tests/Rules/Deprecations/data/call-to-deprecated-static-method.php#L18-L22

we never enter the scope of the deprecated method `Bar::deprecatedFoo()`.